### PR TITLE
Add basic style, colors and header/footer

### PR DESCRIPTION
**Pull request by MrTimscampi**
Starts work on bringing over a similar color scheme, as well as the header and footer links.

image

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -10,71 +10,93 @@ module.exports = {
...
-              label: 'GitHub',
-              href: 'https://github.com/facebook/docusaurus',
+              label: "Check us out on GitHub",
+              href: "https://github.com/jellyfin",
             },
           ],
         },
       ],
-      copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+      copyright: `Site content is licensed <a href="http://creativecommons.org/licenses/by-nd/4.0/">CC-BY-ND-4.0</a>`,
```
Comment (nielsvanvelzen) [inline docusaurus.config.js:99]: There is no license in the repository

File: docusaurus.config.js [hunk 2]
```diff
@@ -10,71 +10,93 @@ module.exports = {
...
+    },
     navbar: {
-      title: 'My Site',
       logo: {
-        alt: 'My Site Logo',
+        alt: 'Jellyfin logo',
         src: 'img/logo.svg',
       },
       items: [
+        { to: "blog", label: "Blog", position: "right" },
```
Comment (nielsvanvelzen) [inline docusaurus.config.js:24]: A lot of your additions use double quotes while the document used only single quotes before.

File: src/pages/clients.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+
+export default function MyReactPage() {
```
Comment (nielsvanvelzen) [inline src/pages/clients.js:4]:
```suggestion
export default function Clients() {
```

File: src/pages/contact.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+
+export default function MyReactPage() {
```
Comment (nielsvanvelzen) [inline src/pages/contact.js:4]:
```suggestion
export default function Contact() {
```

File: src/pages/contribute.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+
+export default function MyReactPage() {
```
Comment (nielsvanvelzen) [inline src/pages/contribute.js:4]:
```suggestion
export default function Contribute() {
```

File: src/pages/downloads.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+
+export default function MyReactPage() {
```
Comment (nielsvanvelzen) [inline src/pages/downloads.js:4]:
```suggestion
export default function Downloads() {
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
Although they are placeholder pages, it would be nice if they used the correct titles

File: docusaurus.config.js (hunk 1 above)
Comment (MrTimscampi) [reply docusaurus.config.js]: Added the license (Copied from jellyfin-blog) to this repository :+1:

File: LICENSE [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (nielsvanvelzen) [inline LICENSE:30]: You're not using the old theme now I suppose?

```suggestion
```

File: LICENSE [hunk 8]
```diff
@@ -0,0 +1,5 @@
+The content of the Jellyfin website is licensed under the
+Creative Commons Attribution-NoDerivatives 4.0 International License.
+
+You may retrieve a copy of it here:
+http://creativecommons.org/licenses/by-nd/4.0/
```
Comment (nielsvanvelzen) [inline LICENSE:5]: At this point you can probably just add the license instead of linking to it 👀

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**